test/e2e-bootstrap/bootstrap_test.go

```diff
@@ -174,6 +174,24 @@ spec:
 			waitForMasterMCs: []string{"99-master-ssh", "99-master-generated-registries"},
 			waitForWorkerMCs: []string{"99-worker-ssh", "99-worker-generated-registries", "99-worker-generated-kubelet"},
 		},
+		{
+			name: "With a container runtime config",
+			manifests: [][]byte{
+				[]byte(`apiVersion: machineconfiguration.openshift.io/v1
+kind: ContainerRuntimeConfig
+metadata:
+  name: cr-pid-limit
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/master: ""
+  containerRuntimeConfig:
+    pidsLimit: 100000
+`),
+			},
+			waitForMasterMCs: []string{"99-master-ssh", "99-master-generated-registries"},
```

Comment:
Shouldn't this test be waiting for the generated continerruntime MC

Reply:
fixed